### DISCUSSION TITLE
Next.js 9.5.0 headers and X-Frame-Options

### DISCUSSION THREAD

**Discussion by ironvale0** [vercel/next.js#15534] [Help]
Hi,
I have `/embed/*` routes with few widgets and they should be iframe available for any origin but all another routes must be protected with `X-Frame-Options:DENY`, how this can be configured with 9.5.0 headers (same issue with [vercel.json headers](https://vercel.com/docs/configuration#project/headers))?

**Comment by ironvale0** [comment]
Any ideas?

**Reply by BramDecuypere** [reply]
Doesn't this answer your question? https://nextjs.org/docs/api-reference/next.config.js/headers
